config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -0,0 +1,52 @@
+periodics:
+- interval: 24h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-compatibility-versions
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: compatibility-version-test
+    description: Uses kubetest & kind to run e2e tests from older (n-1..3) kubernetes releases against a latest kubernetes master components w/ --emulated-version=<n-1..3> set.
+    # TODO(aaron-prindle) route the alert email to a rotation vs individual email
+    testgrid-alert-email: aprindle@google.com
+    testgrid-num-columns-recent: '6'
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/krte:latest-master
+      imagePullPolicy: Always  # pull latest image for canary testing
+      command:
+      - wrapper.sh
+      - bash
+      - -c
+      - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && ${GOPATH}/src/k8s.io/test-infra/experiment/compatibility-versions/e2e-k8s-compatibility-versions.sh
+      env:
+      - name: EMULATED_VERSION
+        value: "1.31"  # TODO(aaron-prindle) FIXME - hardcoded for now
```

Comment:
we might be able to handle this with the config-forker, it only needs to change when we branch the release anyhow

https://github.com/kubernetes/test-infra/tree/master/releng/config-forker <-- run when we introduce a new release branch

Reply:
Ah, I was not aware of the config-forker.  I have created https://github.com/kubernetes/test-infra/issues/33573 to track properly parameterizing the branch/branches to test against.  Currently this won't be an issue as only testing 1.31 is all that is needed for the next ~2 months and I am hoping to get something in to kick the tires on the test.  Lmk if that sg, thanks!

Reply:
Yeah the forker is pretty niche, I think it may be cleaner than alternatives to solve this long term though. Tracking in an issue SGTM